Parsers/ASimAuthentication/Parsers/ASimAuthenticationAADSigninLogs.yaml

```diff
@@ -1,9 +1,9 @@
 Parser:
-  Title: Azure active directory authentication
+  Title: Authentication ASIM parser for Microsoft AAD SignInLogs
   Version: '0.0'
   LastUpdated: June 3, 2021
 Product:
-  Name: Azure SigninLogs
+  Name: Microsoft AAD (SigninLogs)
```

Comment:
"(SigninLogs)" -> "(interactive signin logs)"

Reply:
Changed to interactive authentication